packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.types.ts

```diff
@@ -390,6 +389,7 @@ export interface IContextualMenuItem {
   /**
    * Additional styles to apply to the menu item
    * @defaultvalue undefined
+   * @deprecated in favor of mergeStyles API.
    */
```

Comment:
would be good to further elaborate that this refers to the `styles` prop, since mergeStyles API might not be intuitive to people not that familiar with Fabric's codebase.

Reply:
👍